### ISSUE TITLE
Cannot turn off SLF4J logs

### ISSUE THREAD

**Issue by joshxyzhimself** [elastic/elasticsearch#96652]
### Elasticsearch Version

8.8.0

### Java Version

_bundled_

### OS Version

Ubuntu 22.04

### Problem Description

```env
xpack.security.enabled=true
ingest.geoip.downloader.enabled=false
logger.org.elasticsearch=ERROR
logger.com.azure.core=ERROR
```

Elasticsearch default logs are noisy, how do we remove the following lines on every container run?

```jsonl
{"@timestamp":"2023-06-07T10:22:45.456Z", "log.level": "WARN", "message":"SLF4J: No SLF4J providers were found.", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:45.461Z", "log.level": "WARN", "message":"SLF4J: Defaulting to no-operation (NOP) logger implementation", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:45.462Z", "log.level": "WARN", "message":"SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:45.463Z", "log.level": "WARN", "message":"SLF4J: Class path contains SLF4J bindings targeting slf4j-api versions 1.7.x or earlier.", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:45.464Z", "log.level": "WARN", "message":"SLF4J: Ignoring binding found at [jar:file:///usr/share/elasticsearch/modules/repository-azure/log4j-slf4j-impl-2.19.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:45.464Z", "log.level": "WARN", "message":"SLF4J: See https://www.slf4j.org/codes.html#ignoredBindings for an explanation.", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:51.544Z", "log.level": "WARN", "message":"Jun 07, 2023 10:22:51 AM org.apache.lucene.store.MemorySegmentIndexInputProvider <init>", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:58.448Z", "log.level": "WARN", "message":"SLF4J: No SLF4J providers were found.", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:58.449Z", "log.level": "WARN", "message":"SLF4J: Defaulting to no-operation (NOP) logger implementation", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:58.449Z", "log.level": "WARN", "message":"SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:58.450Z", "log.level": "WARN", "message":"SLF4J: Class path contains SLF4J bindings targeting slf4j-api versions 1.7.x or earlier.", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:58.451Z", "log.level": "WARN", "message":"SLF4J: Ignoring binding found at [jar:file:///usr/share/elasticsearch/modules/x-pack-security/log4j-slf4j-impl-2.19.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
{"@timestamp":"2023-06-07T10:22:58.452Z", "log.level": "WARN", "message":"SLF4J: See https://www.slf4j.org/codes.html#ignoredBindings for an explanation.", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"stderr","elasticsearch.node.name":"1f961da8d955","elasticsearch.cluster.name":"docker-cluster"}
```

### Steps to Reproduce

Run elasticsearch on docker.

**Comment by joshxyzhimself** [comment]
I mean, as a user I just want my logs saved at `/usr/share/elasticsearch/logs` which is quite the default in the previous versions.

**Comment by pgomulka** [comment]
thanks for the bug report @joshxyzhimself 
I agree that those warnings on startup are noisy, we have fixed them and they will be released in 8.9.0
https://github.com/elastic/elasticsearch/pull/95459


> I mean, as a user I just want my logs saved at /usr/share/elasticsearch/logs which is quite the default in the previous versions.

are you running ES with a docker? THen the logs should be outputed into stdout as per standard practice. 
the `/usr/share/elasticsearch/logs` is the directory within the docker container or the directory when running ES from different installation?
see https://www.elastic.co/guide/en/elasticsearch/reference/current/logging.html for more

**Comment by joshxyzhimself** [comment]
It's like this

```yml
volumes:
  - ./docker/volumes/elasticsearch/data:/usr/share/elasticsearch/data
  - ./docker/volumes/elasticsearch/logs:/usr/share/elasticsearch/logs
```

Thank you though, glad it will be less noisy in 8.9.0 thank you thank you!
